tests/Console/Commands/Migrations/MigrateAuth0ToKeycloakClientsTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Console\Commands\Migrations;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Testing\PendingCommand;
+use Symfony\Component\Console\Command\Command;
+use Tests\TestCase;
+
+final class MigrateAuth0ToKeycloakClientsTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private const CLIENT_ID_1 = '27c500be-4cc4-4cb2-97d6-c966a27716c4';
+    private const CLIENT_ID_2 = 'a1905caa-3bca-4ea0-bd60-aa0a13e844a5';
+
+    public function test_invalid_date_format(): void
+    {
+        $this->getPendingCommand('migrate:keycloak', ['updated_at' => 'invalid-date'])
+            ->expectsOutput('Invalid format used for updated at, use the format YYYY-MM-DD')
+            ->assertExitCode(Command::FAILURE);
+    }
+
+    public function test_no_clients_to_migrate(): void
+    {
+        $this->getPendingCommand('migrate:keycloak')
+            ->expectsOutput('No clients found to migrate ')
+            ->assertExitCode(Command::FAILURE);
```

Comment:
I think it is debatable if this is an error.

Reply:
I would also expect this noop to result in success code